lib/sleet/repo.rb

```diff
@@ -2,87 +2,44 @@
 
 module Sleet
   class Repo
```

Comment:
Maybe this isn't really `Repo` maybe it's `Project` which I think fits the CircleCI names better

Reply:
But it also does have the branch here, so not really the project scope... Might leave this for now since I don't know exactly what I want to do